start.sh

```diff
@@ -1,3 +1,5 @@
+#!/bin/bash
```

Comment:
`/bin/sh` may be more future-proof if you don't need bash.  

(Or, if you do need bach, then `/usr/bin/env bash`...)